auth/api/iam/api.go

```diff
@@ -226,13 +236,13 @@ func (r Wrapper) HandleTokenRequest(ctx context.Context, request HandleTokenRequ
 		}
 	case oauth.VpTokenGrantType:
 		// Nuts RFC021 vp_token bearer flow
-		if request.Body.PresentationSubmission == nil || request.Body.Scope == nil || request.Body.Assertion == nil {
+		if request.Body.PresentationSubmission == nil || request.Body.Scope == nil || request.Body.Assertion == nil || request.Body.ClientId == nil {
```

Comment:
derive client_id from subject in URL

Reply:
the `client_id` here is from the requester, the subject in the path is the issuer of the token